plugins/content/bower/index.js

```diff
@@ -837,7 +837,7 @@ function handleUploadedPlugin (req, res, next) {
 
         // first entry should be our target directory
         var packageJson;
-        var canonicalDir = path.join(outputPath, files[0]);
+        var canonicalDir = path.join(outputPath, file.name.replace('.zip', ''));
```

Comment:
This will break if the zip file is not named the same as the contained plugin directory. It's not a normal scenario, but it might be likely.

Consider someone having downloaded the .zip for a plugin twice (resulting in a name like "adapt-contrib-foo(1).zip"), or if it has a timestamp or version number against it.